libs/langchain/langchain/embeddings/openai.py

```diff
@@ -473,7 +469,7 @@ async def _aget_len_safe_embeddings(
         return embeddings
 
     def embed_documents(
-        self, texts: List[str], chunk_size: Optional[int] = 0
+        self, texts: List[str], chunk_size: Optional[int] = None
```

Comment:
the preferred method for passing in chunk_size is actually during init, think this is here for legacy reasons but we should probably add a deprecation warning if it's passed in